Classes/common/Encryption/Keychain/CDTEncryptionKeychainManager+Internal.h

```diff
@@ -0,0 +1,47 @@
+//
+//  CDTEncryptionKeychainManager+Internal.h
+//  CloudantSync
+//
+//  Created by Enrique de la Torre Fernandez on 15/04/2015.
+//  Copyright (c) 2015 IBM Cloudant. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import "CDTEncryptionKeychainManager.h"
+
+/**
+ This category is only for testing purposes. It presents some of the private methods used by
+ CDTEncryptionKeychainManager to complete its job.
+ 
+ Notice that although all these methods are instance methods, none of them modifies the state of a
+ CDTEncryptionKeychainManager instance or writes to disk, however they are easier to override
+ this way.
```

Comment:
It's a bit too easy for this comment to become out of date w.r.t. the implementation.

Do we think they will be overridden? If not, marking them as class methods would imply the stateless nature without the risk of going out of date.